homework_19_11/Task4.java

```diff
@@ -0,0 +1,27 @@
+import static java.lang.Math.*;
+import java.util.Date;
+import java.util.Calendar;
+import java.util.Scanner;
+
+//написать класс который выводит в консоль все елементы командной строки,а если их нету выводит сообщение что их кол-во равно нулю
```

Comment:
Hi.

Please explain what does it mean? 

Now, of course, I can do in another way (with methods), but at that time I was doing it, I knew just this manner.
Please, help. How it would be like?

Reply:
read about command line arguments